CHANGES

```diff
@@ -9,6 +9,26 @@
 
  Changes between 1.1.1 and 3.0.0 [xx XXX xxxx]
 
+  *) Deprecated low level ECDH and ECDSA functions. These include:
+         ECDH_compute_key, ECDSA_SIG_new, d2i_ECDSA_SIG, i2d_ECDSA_SIG,
+         ECDSA_SIG_get0, ECDSA_SIG_get0_r, ECDSA_SIG_get0_s, ECDSA_SIG_set0,
+         ECDSA_do_sign, ECDSA_do_sign_ex, ECDSA_do_verify, ECDSA_sign_setup,
+         ECDSA_sign, ECDSA_sign_ex, ECDSA_verify and ECDSA_size.
+
+     Use of these low level functions has been informally discouraged for a long
+     time.  Instead applications should use the EVP_PKEY_derive(3),
+     EVP_DigestSign(3) and EVP_DigestVerify(3) functions.
+     [Paul Dale]
+
+  *) Deprecated the EC_KEY_METHOD functions. These include:
+         EC_KEY_METHOD_new, EC_KEY_METHOD_free, EC_KEY_METHOD_set_init,
+         EC_KEY_METHOD_set_keygen, EC_KEY_METHOD_set_compute_key,
+         EC_KEY_METHOD_set_sign, EC_KEY_METHOD_set_verify,
+         EC_KEY_METHOD_get_init.  EC_KEY_METHOD_get_keygen.
+         EC_KEY_METHOD_get_compute_key, EC_KEY_METHOD_get_sign and
+         EC_KEY_METHOD_get_verify.
```

Comment:
Needs a statement saying what people should do instead.

Reply:
I've added such a statement, not sure what the best replacement is.

Reply:
The replacement is to write a provider, possibly one built in to the application.